src/test/run-pass/enum-discriminant/arbitrary_enum_discriminant.rs

```diff
@@ -0,0 +1,55 @@
+#![feature(arbitrary_enum_discriminant, const_raw_ptr_deref, test)]
```

Comment:
Move `run-pass/enum-discriminant/*` -> `ui/enum-discriminant` and then put `// run-pass` in each of the moved files.